src/main/java/de/tum/in/www1/artemis/service/ParticipationService.java

```diff
@@ -475,7 +475,7 @@ public StudentParticipation participationForQuizWithResult(QuizExercise quizExer
      * @return List<submission>
      */
     public List<Submission> getSubmissionsWithParticipationId(long participationId) {
```

Comment:
please rename this method as well to reflect that the results + assessor are loaded as well

Reply:
this is important so that other potential users of this method immediately know what exactly is returned (also to realize if data such as the assessor can be exposed or not). This should also be reflected in the JavaDoc of the method